src/FixedPairList.cpp

```diff
@@ -274,6 +282,7 @@ namespace espressopp {
       .def("add", pyAdd)
       .def("size", &FixedPairList::size)
       .def("getBonds",  &FixedPairList::getBonds)
+      .def("remove",  &FixedPairList::remove)
```

Comment:
@MrTheodor I think that `remove` does exactly what it says, i.e. the functionality is correct. As far as I understand standard `connect` and `disconnect` are meaningless here (or at least are meaningless until somebody finds out why they may be used). @govarguz is this correct?

The question for me is whether we want to keep a uniform structure and preserve `connect` and `disconnect` in every module even though in some cases they make no physical sense. I do not have an answer here. Maybe @stuehn or @junghans can tell us what they think or how it supposed to be